app/src/main/java/edu/uc/zhukv/droneradarmap/Weather_Layer/TransparentTileOWM.kt

```diff
@@ -110,7 +110,7 @@ class TransparentTileOWM(private val tileType: String) : TileProvider {
      * `{y}`, and `{zoom}`. An example
      * for an OpenWeatherMap URL would be: http://tile.openweathermap.org/map/precipitation/{zoom}/{x}/{y}.png
      *
-     */        private const val OWM_TILE_URL =
+     */        private const val OPEN_WEATHER_MAP_TILE_URL =
```

Comment:
This variable was called in the code but did not exist, the OWM_TILE_URL variable was not called, so I assumed the name was changed within the code but not at the declaration